functions/datastore/README.md

```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
```

Comment:
Is there not a tutorial for this on devsite? Would it be more valuable to house most of this documentation there?